dice-game-ui-angular/src/app/game/game.component.ts

```diff
@@ -365,4 +369,11 @@ export class GameComponent implements OnInit, AfterViewInit {
     this.renderField();
     return new Rect(minY, minX, maxY - minY + 1, maxX - minX + 1); // inverted axis!!
   }
+
+  ngOnDestroy() {
+    if (this.userServiceSubscription) {
+      this.userServiceSubscription.unsubscribe();
+      this.userServiceSubscription = null;
```

Comment:
Not needed, because no pointers to this variable will exist after the component is destroyed and it will be collected by GC anyway.

Reply:
Do you mean the whole ngOnDestroy method?

Reply:
I mean last line - `this.userServiceSubscription = null;`

Reply:
Got it. Fixed